### PR TITLE
httpxx finder

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: Findhttpxx.cmake [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+# Find headers
+find_path(HTTPXX_INCLUDE_DIR httpxx/Message.hpp
+  HINTS ${HTTPXX_ROOT}/include $ENV{HTTPXX_ROOT}/include
+  ${COMMON_SOURCE_DIR}/httpxx ${CMAKE_SOURCE_DIR}/httpxx
+  /usr/local/include
+  /usr/include)
+
+if(HTTPXX_INCLUDE_DIR)
+    set(HTTPXX_PATH "${HTTPXX_INCLUDE_DIR}/..")
```
Comment (tribal-tec) [inline Findhttpxx.cmake]: 4 space indentation?

File: Findhttpxx.cmake [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+endif()
+
+# Find dynamic libraries
+if(HTTPXX_PATH)
+    set(__libraries httpxx)
+
+    foreach(__library ${__libraries})
+    if(TARGET ${__library})
```
Comment (tribal-tec) [inline Findhttpxx.cmake]: missing indent

File: Findhttpxx.cmake [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  set(_httpxx_output 1)
+endif()
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(httpxx DEFAULT_MSG HTTPXX_LIBRARIES
+  HTTPXX_INCLUDE_DIR)
+
+if(HTTPXX_FOUND)
+  set(HTTPXX_INCLUDE_DIRS ${HTTPXX_INCLUDE_DIR})
+  if(_httpxx_output )
```
Comment (tribal-tec) [inline Findhttpxx.cmake]: too much spacing

File: Findhttpxx.cmake [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+# POSSIBILITY OF SUCH DAMAGE.
+#
+# =============================================================================
+#
+# HTTPXX
+#
+#==============================================================================
+# This module uses the following input variables:
+#
+# HTTPXX_ROOT - Path to the cpp-netlib module
```
Comment (tribal-tec) [inline Findhttpxx.cmake]: C&P error

File: Findhttpxx.cmake [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+# Assume not found.
+set(HTTPXX_FOUND FALSE)
+set(HTTPXX_PATH)
+
+# Find headers
+find_path(HTTPXX_INCLUDE_DIR httpxx/Message.hpp
+  HINTS ${HTTPXX_ROOT}/include $ENV{HTTPXX_ROOT}/include
+  ${COMMON_SOURCE_DIR}/httpxx ${CMAKE_SOURCE_DIR}/httpxx
+  /usr/local/include
```
Comment (tribal-tec) [inline Findhttpxx.cmake]: please add /opt/local/include for homebrew

File: Findhttpxx.cmake [hunk 6]
```diff
@@ -0,0 +1,98 @@
+#
```
Comment (tribal-tec) [inline Findhttpxx.cmake]: they don't provide package config as a CMake project? Push this upstream?

File: Findhttpxx.cmake (hunk 1 above)
Comment (eile) [inline Findhttpxx.cmake]: c&p, will fix

File: Findhttpxx.cmake (hunk 3 above)
Comment (eile) [inline Findhttpxx.cmake]: all that stuff is C&P from cppnetlib:-/

File: Findhttpxx.cmake (hunk 6 above)
Comment (eile) [inline Findhttpxx.cmake]: No, and it's not easily done. Was my first thought too. Maybe in a second iteration.

**Comment by eile** [comment]
All done.

**Pull request merged**